sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -268,7 +237,6 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
       .orderBy("status", "DESC")
       .addOrderBy("assessment.submittedDate", "DESC")
       .getRawAndEntities();
-
     return mapFromRawAndEntities<AssessmentHistory>(queryResult, "status");
```

Comment:
The `mapFromRawAndEntities` was needed here just because of the `CASE WHEN`, unless I am missing something.
Can we use the regular `getMany` now?